qemu/tests/numa_stress.py

```diff
@@ -87,6 +87,8 @@ def run(test, params, env):
     most_used_node, memory_used = max_mem_map_node(host_numa_node, qemu_pid)
 
     for test_round in range(test_count):
+        if os.path.isfile(memory_file):
```

Comment:
@yanan-fu And yes, the comment 4 in the bz is not a bug literally. It's about the host env problem.  I suggest to run the case with suitable memory.  Thanks for your test!